test/unit/on-attach.spec.js

```diff
@@ -749,4 +771,64 @@ describe('onAttach', function() {
       });
     });
   });
+
+  describe('when showing an empty CollectionView', function() {
```

Comment:
Could we add some tests where the `CollectionView` is also attached to a region that's atached to the document?